Aseprite.js

```diff
@@ -271,12 +303,14 @@ class Aseprite {
     const opacity = this.readNextByte();
     this.skipBytes(3);
     const name = this.readNextString();
+    const tilesetIndex = (type == 2) ? this.readNextDWord() : undefined;
     this.layers.push({ flags,
       type,
       layerChildLevel,
       blendMode,
       opacity,
-      name});
+      name,
+      tilesetIndex});
```

Comment:
I'm not a fan of adding a property that could be `undefined`.

We could create an object at the top of this function and then add each property in it instead of creating variables and resolve it that way.

Reply:
Fixed in b11bcb1.